Fitbit.Portable/IFitbitClient.cs

```diff
@@ -37,9 +35,12 @@ public interface IFitbitClient
         Task<WaterData> GetWaterAsync(DateTime date);
         Task<WaterLog> LogWaterAsync(DateTime date, WaterLog log);
         Task DeleteWaterLogAsync(long logId);
-
         Task<List<ApiSubscription>> GetSubscriptionsAsync();
         Task<ApiSubscription> AddSubscriptionAsync(APICollectionType apiCollectionType, string uniqueSubscriptionId, string subscriberId = default(string));
         Task DeleteSubscriptionAsync(APICollectionType collection, string uniqueSubscriptionId, string subscriberId = null);
+        Task<ActivityLog> LogActivityAsync(ActivityLog model);
+        Task<HeartActivitiesTimeSeries> GetHeartRateTimeSeries(DateTime date, DateRangePeriod dateRangePeriod, string userId = "-");
+        Task<HeartActivitiesIntraday> GetHeartRateIntraday(DateTime date, HeartRateResolution resolution, string userId = "-");
```

Comment:
GetHeartRateIntradayAsync to keep with the pattern

Reply:
https://github.com/aarondcoleman/Fitbit.NET/blob/079c870529030d1775f08653569115eaaa6b67c8/Fitbit.Portable/FitbitClient.cs#L422